lib/Grid.js

```diff
@@ -6,7 +6,15 @@ const autoRows = ({ minRowHeight = "20px" }) => `minmax(${minRowHeight}, auto)`;
 const columns = ({ columns = 12 }) =>
   typeof columns === "number" ? `repeat(${columns}, 1fr)` : columns;
 
-const gap = ({ gap = "8px" }) => `${gap} ${gap}`;
+const gap = ({ gap = "8px" }) => {
+  if (gap.match(/\s/)) {
+    if (gap.split(" ").length - 1 > 1) {
```

Comment:
I wouldn't worry about throwing an error here, you can delete the inner `if` block.